packages/@react-aria/grid/src/useGridCell.ts

```diff
@@ -35,15 +38,16 @@ export function useGridCell<T, C extends GridCollection<T>>(props: GridCellProps
     node,
     ref,
     isVirtualized,
-    isDisabled
+    isDisabled,
+    focusMode = 'child'
   } = props;
 
   // Handles focusing the cell. If there is a focusable child,
   // it is focused, otherwise the cell itself is focused.
   let focus = () => {
     let treeWalker = getFocusableTreeWalker(ref.current);
     let focusable = treeWalker.firstChild() as HTMLElement;
```

Comment:
We may want to do something like this so that if you had multiple focusable items in a cell, navigating backward from the next cell into the previous cell focuses the last focusable child instead of the first.

```tsx
let focusable = ref.current.compareDocumentPosition(document.activeElement) & Node.DOCUMENT_POSITION_FOLLOWING
      ? treeWalker.lastChild()
      : treeWalker.firstChild() as HTMLElement;
```